app/assets/javascripts/components/option-select.js

```diff
@@ -56,21 +59,27 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
 
     // Attach listener to update checked count
-    this.$optionSelect.on('change', "input[type='checkbox']", this.updateCheckedCount.bind(this))
+    this.$optionsContainer.querySelector('.gem-c-checkboxes__list').addEventListener('change', function () {
```

Comment:
It seems unlikely that this element would not exist - I would think an `OptionSelect` without options is not very useful.  Having said that, we did have to add some additional elements inside the top-level element down-to (and including) the `div .gem-c-checkboxes__list` in the `html` element in the tests.  Not to make this work, but because we wanted to add an event listener on the checkboxes parent element.  Without adding these additional elements the tests were breaking - there is more detail on this [here](https://github.com/alphagov/finder-frontend/pull/2525#discussion_r654478497).

Reply:
There is definitely something odd going on here...

Reply:
Updated to use `bind`.  See [here](https://github.com/alphagov/finder-frontend/pull/2525#discussion_r657186711)